src/main/resources/com/microsoft/azure/vmagent/AzureVMAgentTemplate/help-galleryImageDefinition.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Images are defined within a gallery and carry information about the image and requirements for using it internally. This includes whether the image is Windows or Linux, release notes, and minimum and maximum memory requirements. It is a definition of a type of image.
```

Comment:
is this from azure doc?

Reply:
Yes, https://docs.microsoft.com/en-us/azure/virtual-machines/windows/shared-image-galleries.